packages/blocks/src/api/raw-handling/list-reducer.js

```diff
@@ -8,7 +8,9 @@ function isList( node ) {
 }
 
 function shallowTextContent( element ) {
-	return [ ...element.childNodes ]
+	// On mobile DOM implmentation NodeList is not spreadable so we need to convert it to an array.
```

Comment:
Does Array.from not work?

Reply:
Just tested and it works fine! Thanks for the suggestion.